### PR TITLE
feat(types): update all the types to match the latest changes in angular

### PR DESCRIPTION
**Pull request by stevemao**
https://github.com/angular/angular/blob/master/CONTRIBUTING.md#type

### CODE REVIEW

File: index.json [hunk 1]
```diff
@@ -10,19 +10,25 @@
...
+    },
+    "build": {
+      "description": "Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)"
+    },
+    "ci": {
+      "description": "Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)"
     },
     "chore": {
-      "description": "Changes to the build process or auxiliary tools and libraries such as documentation generation"
+      "description": "Other changes that don't modify src or test files"
```
Comment (stevemao) [inline index.json]: Looks like `chore` is separated into `build` and `ci`

Comment (stevemao) [inline index.json]: Looks like `chore` is separated into `build` and `ci`

**Comment by adjohnson916** [comment]
@stevemao Thanks, I'll publish a new version soon.

**Comment by adjohnson916** [comment]
@jimthedev Do you mind if `cz-conventional-changelog` inherits these updates with its 2.x dependency, or need me to release as 3.x?

**Comment by stevemao** [comment]
I think a minor bump is sufficient. But to be save 3.x also makes sense. cc @LinusU and @ jimthedev

**Comment by LinusU** [comment]
Since it doesn't remove any types I think it's safe to release as a minor bump

**Comment by jimthedev** [comment]
agreed.

**Comment by adjohnson916** [comment]
Released as `v2.1.0`.

**Pull request merged**